### ISSUE TITLE
Default Bounding Box Priors

### ISSUE THREAD

**Issue by anujdutt9** [uvipen/SSD-pytorch#8]
Hi, 

Thanks for your implementation. This makes understanding the model much more easier.
I did notice one thing though. In the paper, the authors mention about defining default bounding boxes as priors and define different aspect ratios and scales for the same as well. Then during training they calculate the offset values between the priors with the overlap greater than a threshold w.r.t the ground truth bounding boxes and use that to calculate the localization loss.
Although, this implementation trains and works well, but I couldn't find the use of default bounding boxes anywhere.
Am I missing something here?

Thanks.
